apps/blockchain/lib/blockchain/account.ex

```diff
@@ -514,7 +514,7 @@ defmodule Blockchain.Account do
       iex> Blockchain.Account.get_storage(updated_state, <<01::160>>, 5)
       {:ok, 9}
   """
-  @spec put_storage(EVM.state(), EVM.address(), integer(), integer()) :: t
+  @spec put_storage(EVM.state(), EVM.address(), integer(), integer()) :: EVM.state | { EVM.state, t, t }
```

Comment:
Something is wrong with this typespec. Why should the method return a tuple with duplicate items? ```{ EVM.state, t, t }```
I think it returns just `EVM.state`